docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md

```diff
@@ -21,9 +23,9 @@ The diagram presents monitoring components and the way they interact with one an
 ![Monitoring components](./assets/obsv-monitoring-architecture.svg)
 
 
-1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a Prometheus instance, manages its deployment, and provides configuration for it. It also deploys Alertmanager and operates ServiceMonitor custom resources that specify monitoring definitions for groups of services.
+1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a **Prometheus** instance, manages its deployment, and provides configuration for it. It also deploys **Alertmanager** and operates **ServiceMonitor** custom resources that specify monitoring definitions for groups of services.
```

Comment:
One would operate an active component only. Here it is a resource, so managing is the better term
```suggestion
1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a **Prometheus** instance, manages its deployment, and provides configuration for it. It also deploys **Alertmanager** and manages **ServiceMonitor** custom resources that specify monitoring definitions for groups of services.
```